app/src/main/java/de/westnordost/streetcomplete/quests/sidewalk_surface/AddSideWalkSurfaceForm.java

```diff
@@ -0,0 +1,101 @@
+package de.westnordost.streetcomplete.quests.sidewalk_surface;
+
+import de.westnordost.streetcomplete.quests.AbstractQuestFormAnswerFragment;
+
+import android.os.Bundle;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import java.util.Arrays;
+
+import de.westnordost.streetcomplete.R;
+import de.westnordost.streetcomplete.quests.AbstractQuestFormAnswerFragment;
+import de.westnordost.streetcomplete.quests.road_surface.AddRoadSurface;
+import de.westnordost.streetcomplete.view.GroupedImageSelectAdapter;
+
+/**
+ * Created by Neelam Purswani on 14-01-2018.
+ */
+
+public class AddSideWalkSurfaceForm extends AbstractQuestFormAnswerFragment {
+    public static final String SURFACE = "surface";
+
+    private final AddSideWalkSurfaceForm.Surface[] SURFACES = new AddSideWalkSurfaceForm.Surface[] {
```

Comment:
When testing, make sure that the answer options make sense for footways, and that the images aren't of things that are obviously streets.